packages/extension-ui/src/components/Address.tsx

```diff
@@ -221,7 +221,7 @@ function Address ({ actions, address, children, className, genesisHash, isExtern
                   : undefined
               }
             >
-              {chain.name.replace('Relay Chain', '')}
+              {chain.name.replace(' Relay Chain', '')}
```

Comment:
:tada: my test caught this tiny bug. Kusama and Polkadot labels actually had a white space at the front.

Reply:
:)